samples/migrations/qdrant_migration_test.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+from typing import Sequence
+
+import pytest
+import pytest_asyncio
+from langchain_core.documents import Document
+from langchain_core.embeddings import FakeEmbeddings
+from langchain_qdrant import QdrantVectorStore
+from migrate_qdrant_vectorstore_to_alloydb import main
+from qdrant_client import QdrantClient
+from qdrant_client.http.models import Distance, VectorParams
+from sqlalchemy import text
+from sqlalchemy.engine.row import RowMapping
+
+from langchain_google_alloydb_pg import AlloyDBEngine
+
+DEFAULT_TABLE = "test_qdrant_migration" + str(uuid.uuid4())
+EMBEDDING_SERVICE = FakeEmbeddings(size=768)
+PERSISTENT_DB_PATH = "./qdrant_data"
+
+
+def get_env_var(key: str, desc: str) -> str:
+    v = os.environ.get(key)
+    if v is None:
+        raise ValueError(f"Must set env var {key} to: {desc}")
+    return v
+
+
+async def aexecute(
+    engine: AlloyDBEngine,
+    query: str,
+) -> None:
+    async def run(engine, query):
+        async with engine._pool.connect() as conn:
+            await conn.execute(text(query))
+            await conn.commit()
+
+    await engine._run_as_async(run(engine, query))
+
+
+def create_qdrant_collection(collection_name):
+    client = QdrantClient(path=PERSISTENT_DB_PATH)
+    client.delete_collection(
```

Comment:
Remove this or Add a note suggesting delete only helps avoid pre-existing collections